src/seedu/addressbook/commands/DeleteExamCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyExam;
+import seedu.addressbook.data.person.UniqueExamList.ExamNotFoundException;
+
+/**
+ * Deletes a exam identified using its last displayed index from the exam book.
+ */
+public class DeleteExamCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteexam";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Deletes the exam identified by the index number used in the last exams listing.\n\t"
+            + "Parameters: INDEX\n\t"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_EXAM_SUCCESS = "Deleted Exam: %1$s";
+
+    /**
+     * Constructor used for Privileges
+     * Command constructed has no functionality
+     * */
+    public DeleteExamCommand() {
+    }
+
+    public DeleteExamCommand(int targetVisibleIndex) {
+        super(targetVisibleIndex);
+    }
+
+    @Override
+    public CommandResult execute() {
+        try {
+            final ReadOnlyExam target = getTargetExam();
+            examBook.removeExam(target);
+            return new CommandResult(String.format(MESSAGE_DELETE_EXAM_SUCCESS, target));
+        } catch (IndexOutOfBoundsException ie) {
+            return new CommandResult(Messages.MESSAGE_INVALID_EXAM_DISPLAYED_INDEX);
+        } catch (ExamNotFoundException enfe) {
+            return new CommandResult(Messages.MESSAGE_EXAM_NOT_IN_EXAMBOOK);
+        }
+    }
+
+    @Override
+    public boolean isMutating() {
+        return true;
+    }
+
+    @Override
+    public Category getCategory() {
+        return Category.DETAILS;
```

Comment:
Isnt this method for category of different privileges?

Reply:
No no no. 
getCategory() is for the help command to group the commands of the same category together when showing the help list.
The one for restricting commands base on privilege is under the "Users" class/subclasses.
Each subclass have it's own list of commands that it can newly access to

Reply:
So the point of categorizing under "EXAMS" is just to let it know whether this command is exam-related?

Reply:
Yea, so that, specifically, the help command knows its category and group them together. In the original AB, it was ordered by the way you put it into the list. But this one, it automatically group commands of the same command together, then sort alphabetically

Reply:
I thought the sort was based on increasing privileges. If another category was added the list for HelpCommand would be different.

Reply:
I was thinking of doing that at first, but I think that would be weird to have it cycle through the categories rather than having commands of similar feature in the same place

Reply:
Did not know what you were doing as it was not written down :). I will add a exam category.

Reply:
The HelpCommand has to be worked on to be more clearer. I will post an issue for this.

Reply:
> Did not know what you were doing as it was not written down :). I will add a exam category.

Well to be fair, I did explain it in the group

Reply:
Created in #95